contact.html

```diff
@@ -4,8 +4,82 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Contact</title>
+    <link rel="stylesheet" href="index.css">
 </head>
 <body>
+    <header>
+        <div>
+            Alex Reber's Site!
+        </div>
+    </header>
+    <nav>
+        <div>
+            <a href="index.html">Home</a> 
+            <a href="contact.html">Contact</a> 
```

Comment:
This link, on the 'Contact' page, will send a request to the web server forcing the browser to reload the page with _**no**_ benefit.  Consider an approach which would allow 'Contact' to be displayed as is, without resulting in the "reloading" behavior.